requirements/travis.txt

```diff
@@ -4,22 +4,47 @@
 #
 #    make upgrade
```

Comment:
Do we still run tests in Travis / should we file a ticket to update that?

Reply:
It's being used in [docker-compose-travis.yml](https://github.com/edx/edx-notes-api/blob/master/.travis/docker-compose-travis.yml)

Reply:
Should I merge this PR?